src/goto-programs/goto_program.h

```diff
@@ -1078,6 +1079,17 @@ class goto_programt
       nil_exprt(),
       {});
   }
+
+  // Returns an assertion smaller < larger. 
+  // This is necessary for the implementation of loop variants. 
+  static instructiont make_inequality(
```

Comment:
This function is only ever called once, so why don't we just inline its definition there? It's a single line anyway ...

Reply:
Good point. Yes, I will fix it.